services/venice-controller/src/main/java/com/linkedin/venice/controller/ServiceDiscoveryAnnouncerHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.linkedin.venice.controller;
+
+import com.linkedin.venice.servicediscovery.ServiceDiscoveryAnnouncer;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.BlockingQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class ServiceDiscoveryAnnouncerHelper {
```

Comment:
Instead of having a `ServiceDiscoveryAnnouncerHelper` class, can we add a new `ServiceDiscoveryAnnouncer` implementation that wraps a list of other `ServiceDiscoveryAnnouncer` objects?

Like `RetryingServiceDiscoveryAnnouncer` or `AsyncRetryingServiceDiscoveryAnnouncer`?

Let's also move it to the same folder as `ServiceDiscoveryAnnouncer`. There is nothing Controller-specific in this class